addonscript/api.py

```diff
@@ -0,0 +1,50 @@
+"""API helpers for addonscript."""
+
+from addonscript.utils import get_api_url, amo_put, amo_get, amo_download
+from addonscript.task import get_channel
+
+# https://addons-server.readthedocs.io/en/latest/topics/api/signing.html#uploading-a-version
+UPLOAD_VERSION = "api/v3/addons/{id}/versions/{version}/"
+
+# https://addons-server.readthedocs.io/en/latest/topics/api/signing.html#checking-the-status-of-your-upload
+UPLOAD_STATUS = "api/v3/addons/{id}/versions/{version}/upload/{upload_pk}/"
```

Comment:
The api is versioned, and it looks like the version is *used* (v3, as opposed to some of our perma-v1's). We may want to version our task definition + api interface, so we can drop in v4 support in parallel when amo adds v4.

Not a blocker, but is potentially an area for future work.